frontend/app/note/page.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";
+import { PageHeader } from "@/lib/components/PageHeader/PageHeader";
+
+import styles from "./page.module.scss";
+
+const Note = (): JSX.Element => {
+  return (
+    <div className={styles.main_container}>
+      <div className={styles.page_header}>
+        <PageHeader iconName="note" label="My Notes" buttons={[]} />
```

Comment:
Rename My Notes by Notetaker :) 
And for the icon, good initiative, but maybe a pen icon is better :)